contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,775 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> PostHog is not currently SOC 2 certified, but is currently being audited in order to complete this certification. 
+
+## Frequently asked questions
+
+How much does LogRocket cost?
+How much does PostHog cost?
+How do LogRocket and PostHog compare?
+Do LogRocket and PostHog offer free trials?
+Is PostHog or LogRocket more popular?
+
+Got another question? You can [ask the PostHog team anything you want](/questions)!
+
+### How much does LogRocket cost?
+
+LogRocket has a complex pricing structure which isn’t fully transparent. Although it offers a free trial, this is limited to only 1,000 sessions per month. Beyond this, subsequent product tiers begin at $99 USD per month, depending on usage. This version does not include access to many features, such as product analytics. 
+
+More feature-rich product tiers begin at $550 USD per month, but increase depending on how many session are captured — and the per session cost is not clear. Pricing for the full version of LogRocket is not publicly available. 
+
+### How much does PostHog cost?
+
+PostHog has transparent pricing which is based only around the amount of usage. It’s free to get started and completely free for the first 1 million events and 15,000 sessions captured every month.
+
+After this free monthly allowance, PostHog charges $0.00045/event and $0.005/recording, Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing. 
+
+### How do LogRocket and PostHog compare?
+
+PostHog and LogRocket contain some similar features, but are ultimately intended for very different needs and types of users. 
+
+LogRocket is built primarily for front-end developers who need to monitor performance and gather usage information. It has robust session recording and error monitoring tools to accomplish this. 
+
+PostHog is built for engineers and developers in a more general capacity and offers a wider range of tools, which are also suitable for product and growth teams. It also offers robust session recording tools and the ability to track errors, but also includes feature flagging, experimentation and more. 
+
+### Do LogRocket and PostHog offer free trials?
+
+LogRocket offers a free version of its basic product, which is limited to only 1,000 session captures per month. 14-day free trials are also available for other product tiers, after which users must pay to continue usage. 
+
+With PostHog, it’s free to get started and all users get their first 1 million events and 15,000 sessions for free, every month. Billing limits can be used to keep usage beneath this allowance, enabling you to use PostHog for free indefinitely. 
+
+### Is PostHog or LogRocket more popular?
```

Comment:
I don't really like the "popularity" framing – feels a bit superficial. Surely just to ask "Who uses PostHog? / Who uses LogRocket?" You can cover the same stuff to add some social proof.